src/main/java/net/pms/dlna/MediaMonitor.java

```diff
@@ -206,49 +206,44 @@ public void stopped(DLNAResource res) {
 						LOGGER.debug("An error occurred when dumping monitor file: " + e);
 					}
 
-					File watchedFile = new File(rf.getFile().getAbsolutePath());
+					File playedFile = new File(rf.getFile().getAbsolutePath());
 
 					if (fullyPlayedAction == 3) {
 						// Move the video to a different folder
-						String newDirectory = configuration.getFullyPlayedOutputDirectory();
-						if (!newDirectory.endsWith("\\")) {
-							newDirectory += "\\\\";
-						}
+						String newDirectory = FileUtil.appendPathSeparator(configuration.getFullyPlayedOutputDirectory());
```

Comment:
I made this method just for cases like this, and it takes into account different OS'es. What you made here would only work on Windows.